app/models/ncr/work_order.rb

```diff
@@ -116,8 +116,8 @@ def approving_official
 
     # the highest approver on the stack, pending preferred if status indicates
     def current_approver_email_address
-      if self.individual_approvals.pending.first
-        self.individual_approvals.pending.first.user.email_address
+      if self.pending?
+        self.individual_approvals.where(status: 'actionable').first.user.email_address
```

Comment:
Checking for actionable here instead of grabbing the first pending approval. I tested this with proposal ID 265 on staging.